slf4j/src/main/java/com/google/common/flogger/backend/slf4j/SLF4JBackendFactory.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2018 The Flogger Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.common.flogger.backend.slf4j;
+
+import com.google.common.flogger.backend.LoggerBackend;
+import com.google.common.flogger.backend.system.BackendFactory;
+import org.slf4j.LoggerFactory;
+
+/**
+ * BackendFactory for SLF4J.
+ *
+ * <p>To configure this backend for Flogger set the following system property (also see {@link
+ * com.google.common.flogger.backend.system.DefaultPlatform}):
+ *
+ * <ul>
+ * <li>{@code flogger.backend_factory=
+ * com.google.common.flogger.backend.slf4j.SLF4JBackendFactory#getInstance}.
+ * </ul>
+ */
+public final class SLF4JBackendFactory extends BackendFactory {
```

Comment:
Google Java style makes all acronyms in UpperCamelCase. So this should be Slf4JBackendFactory (or maybe Slf4jBackendFactory, I'm not actually sure). @kevinb9n thoughts?

Reply:
I'd bet, it must be lowecase "j" as there's no "SLF" and no "J" acronym there. While "SLF" alone makes some sense, the thing involved is "SLF4J". Otherwise, you could write "HtTP" as "Hypertext" makes some sense, too.